### DISCUSSION TITLE
how to disable/hide the "Sitemap"

### DISCUSSION THREAD

**Discussion by xiao-zong** [academicpages/academicpages.github.io#2104] [Q&A] [answered]
<img width="319" alt="image" src="https://github.com/academicpages/academicpages.github.io/assets/85374952/d20bc727-4273-4dc0-9d61-85c0e12e56e3">

**Comment by rjzupkoii** [comment, marked as answer]
I've updated the [Wiki with information on this](https://github.com/academicpages/academicpages.github.io/wiki/Customizing-the-layout-of-your-site#removing-the-sitemap), but the Sitemap can be removed by editing the file `_includes/footer/custom.html` and deleting the following line:

```html
<a href="/sitemap/">Sitemap</a>
```

**Reply by xiao-zong** [reply]
Thank you for your kind assistance!

**Comment by dongyls** [comment]
Just to add, the above answer is about removing "Sitemap". To remove "FEED", you need to edit `atom_feed` in `_config.yml`.
